sdk/servicebus/azure-servicebus/azure/servicebus/aio/async_client.py

```diff
@@ -53,13 +53,15 @@ class ServiceBusClient(mixins.ServiceBusMixin):
 
     def __init__(self, *, service_namespace=None, host_base=SERVICE_BUS_HOST_BASE,
                  shared_access_key_name=None, shared_access_key_value=None, loop=None,
+                 transport_type=None,
```

Comment:
Docstrings please :)

Reply:
I hope I did that correctly, be gentle if not

Reply:
LGTM :)